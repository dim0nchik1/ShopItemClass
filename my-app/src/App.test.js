import { render, screen } from '@testing-library/react';
import App from './App';

test('test react', () => {
  expect(render(<App />)).toMatchSnapshot();
})
