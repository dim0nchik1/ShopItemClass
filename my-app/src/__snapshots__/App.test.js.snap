// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test react 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <div
          class="background-element"
        />
        <div
          class="highlight-window"
        >
          <div
            class="highlight-overlay"
          />
        </div>
        <div
          class="window"
        >
          <div
            class="main-content"
          >
            <h2>
              Tiger of Sweden
            </h2>
            <h1>
              Leonard coat
            </h1>
            <h3>
              Minimalistic coat in cotton-blend
            </h3>
            <div
              class="description"
            >
              Men's minimalistic overcoat in cotton-blend. Features a stand-up collar, concealed front closure and single back vent. Slim fit with clean, straight shape. Above-knee length.
            </div>
            <div
              class="highlight-window mobile"
            >
              <div
                class="highlight-overlay"
              />
            </div>
            <div
              class="divider"
            />
            <div
              class="purchase-info"
            >
              <div
                class="price"
              >
                £
                399.00
              </div>
              <button>
                Добавить в корзину
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <div
        class="background-element"
      />
      <div
        class="highlight-window"
      >
        <div
          class="highlight-overlay"
        />
      </div>
      <div
        class="window"
      >
        <div
          class="main-content"
        >
          <h2>
            Tiger of Sweden
          </h2>
          <h1>
            Leonard coat
          </h1>
          <h3>
            Minimalistic coat in cotton-blend
          </h3>
          <div
            class="description"
          >
            Men's minimalistic overcoat in cotton-blend. Features a stand-up collar, concealed front closure and single back vent. Slim fit with clean, straight shape. Above-knee length.
          </div>
          <div
            class="highlight-window mobile"
          >
            <div
              class="highlight-overlay"
            />
          </div>
          <div
            class="divider"
          />
          <div
            class="purchase-info"
          >
            <div
              class="price"
            >
              £
              399.00
            </div>
            <button>
              Добавить в корзину
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
